bitbots_hcm/src/bitbots_hcm/hardware_control_manager.py

```diff
@@ -14,7 +14,7 @@
 from humanoid_league_speaker.speaker import speak
 from bitbots_msgs.msg import FootPressure
 
-from bitbots_msgs.msg import JointCommand
+from bitbots_msgs.msg import JointCommand, JointTorque
```

Comment:
If line 50 is not use, this should be commented out as well

Reply:
or delete line 50